### PR TITLE
[imaging_browser] Updated test plan

### LINKED ISSUE #6436: [imaging_browser] Test plan does not test widgets
The imaging browser test plan does not include tests for the widgets that the module registers on the candidate dashboard.

### PR DESCRIPTION
**Pull request by pierre-p-s**
## Brief summary of changes

- Updated the test plan to test the new widgets in the new Access profile module

#### Link(s) to related issue(s)

* Resolves #6436

### CODE REVIEW

File: modules/imaging_browser/test/TestPlan.md [hunk 1]
```diff
@@ -41,3 +41,8 @@ prepended with the Patient Name.
 25. On the view session page, click the button "Visit Level Feedback".
 26. The entries in this dialog should be editable when a user has the permission `imaging_browser_qc`.
 27. Try editing comments (adding new ones, deleting old ones). Click Save and ensure the data is saved.
+
+### Test the Candidate Dashboard widget
+28. Go to the candidate dashboard in the candidate module and check the Imaging QC Summary.
```
Comment (christinerogers) [inline modules/imaging_browser/test/TestPlan.md:46]:
```suggestion
28. Go to the candidate dashboard in the candidate module and check the Imaging QC Summary widget.
```

File: modules/imaging_browser/test/TestPlan.md [hunk 2]
```diff
@@ -41,3 +41,8 @@ prepended with the Patient Name.
...
+    - For each visit, check that the QC status corresponds to the one in the Imaging Browser Module.
+    - Check that all the files from the Imaging Browser Module appears in the widget for all the visits.
```
Comment (christinerogers) [inline modules/imaging_browser/test/TestPlan.md:48]:
```suggestion
    - For a few candidate/visits, check that all the files from the Imaging Browser Module appear in the widget.
```

File: modules/imaging_browser/test/TestPlan.md [hunk 3]
```diff
@@ -41,3 +41,8 @@ prepended with the Patient Name.
 25. On the view session page, click the button "Visit Level Feedback".
 26. The entries in this dialog should be editable when a user has the permission `imaging_browser_qc`.
 27. Try editing comments (adding new ones, deleting old ones). Click Save and ensure the data is saved.
+
+### Test the Candidate Dashboard widget
+28. Go to the candidate dashboard in the candidate module and check the Imaging QC Summary.
+    - For each visit, check that the QC status corresponds to the one in the Imaging Browser Module.
```
Comment (christinerogers) [inline modules/imaging_browser/test/TestPlan.md:47]:
```suggestion
    - For each visit, check that the QC status displayed matches the Imaging Browser Module.
```

**Comment by pierre-p-s** [comment]
> Thanks for your updates.
> Suggested some clarifications and typo fixes.
> 
> Not sure but are there no other features in the Widget that are important like--- How many Scans are waiting to be QC'd (or just new Scans)?

From what I saw the widget gives a graph of the Scans for each visits. Depending on the color, it displays if the QC passed, failed or 'other'.

File: modules/imaging_browser/test/TestPlan.md [hunk 4]
```diff
@@ -41,3 +41,8 @@ prepended with the Patient Name.
 25. On the view session page, click the button "Visit Level Feedback".
 26. The entries in this dialog should be editable when a user has the permission `imaging_browser_qc`.
 27. Try editing comments (adding new ones, deleting old ones). Click Save and ensure the data is saved.
+
+### Test the Candidate Dashboard widget
+28. Go to the candidate dashboard in the candidate module and check the Imaging QC Summary widget.
```
Comment (driusan) [inline modules/imaging_browser/test/TestPlan.md:46]: It seems to be missing tests for the user functionality that is described in the widget itself (ie hovering, clicking..)

File: modules/imaging_browser/test/TestPlan.md [hunk 5]
```diff
@@ -41,3 +41,8 @@ prepended with the Patient Name.
...
+    - For each visit, check that the QC status displayed matches the Imaging Browser Module.
```
Comment (driusan) [inline modules/imaging_browser/test/TestPlan.md:47]: This is missing tests for the breakdowns by status

**Pull request merged**